assets/js/gamepage.js

```diff
@@ -51,4 +45,12 @@ centerThemeCard.on('click', function(event) {
     } else {
       console.log("Computer wins");
     }
-}
\ No newline at end of file
+}
```

Comment:
we don't need these additional lines every time the page loads. The initialize funtion in main.js which is loaded before gamepage.js takes care of getting a new deck. if you want to add a call to the shuffle function inside of initialize right after the call to renderBottomRow() on line 47 in main.js

Reply:
Once the initialize function is called, the bottom row is rendered with 52 cards and when we call the shuffle or drawCard functions there will be no card to draw since all 52 cards are used up by the bottom row. That is the reason why we need to get a new deck every time we play a new game. 


If we call drawCards(2) now, we will get an error that no card is available to be drawn. I hope this answers your concern.

Reply:
I have made the change as you mentioned and shuffled the card using the same deck after the bottom row is rendered. Since the renderBottomRow is an async function, we cannot add this shuffle after line 47 in main.js. I hope this works.